### ISSUE TITLE
Incorporate bug fixes from WISE into Prometheus

### ISSUE THREAD

**Issue by RobBryce** [WISE-Developers/Prometheus-EOL-Issues#16]
WISE received a lot of development recently, and inadvertently, some bug fixes (typically edge cases for failures during loading an FGMJ file) were not applied to Prometheus.

Prometheus and WISE are now completely separate w/r to repositories, repository layout, and history, so fixes need to be manually incorporated.

**Comment by spydmobile** [comment]
Great point, and from now on, if you fix something in wise, that could impact PEOL, you need to make an equivalent issue for Prometheus EOL so you dont have to make work and go back later to find what changes to bring forward.

**Comment by RobBryce** [comment]
Understood, but we were working against an expired clock on the WISE deliverable.

**Comment by spydmobile** [comment]
> Understood, but we were working against an expired clock on the WISE deliverable.

yup, not nitpicking what you did, just laying out our direction moving forward ;-)
